brooklyn-tosca-transformer/src/main/java/io/cloudsoft/tosca/a4c/brooklyn/PolicyDecorator.java

```diff
@@ -0,0 +1,10 @@
+package io.cloudsoft.tosca.a4c.brooklyn;
+
+import java.util.Map;
+import java.util.Set;
+
+import com.google.common.base.Optional;
+
+public interface PolicyDecorator {
```

Comment:
Either name this differently or include documentation to distinguish between Brooklyn's policies and TOSCA's policies.